gamemodes/terrortown/gamemode/client/cl_help.lua

```diff
@@ -275,11 +289,23 @@ end
 function HELPSCRN:BuildContentArea()
 	local parent = self.parent
 
-	if not IsValid(parent) then return end
+	if not IsValid(parent) then
+		return
+	end
 
 	---
 	-- @realm client
-	if hook.Run("TTT2OnHelpSubmenuClear", parent, self.currentMenuId, self.lastMenuData, self.submenuClass) == false then return end
+	if
+		hook.Run(
+			"TTT2OnHelpSubmenuClear",
+			parent,
+			self.currentMenuId,
+			self.lastMenuData,
+			self.submenuClass
+		) == false
```

Comment:
Just as a reference, here is the hook in the style I suggested earlier

Reply:
this is a single function and not a variable declaration as in most other cases you commented on

Reply:
but my point is, that it is inconsitent. Here it is done like this, which I quite like. In another part of the code it is done like this which is really weird imho:
![image](https://github.com/TTT-2/TTT2/assets/13639408/20cdd5be-6bd8-4ada-b1bf-8d60ad5ef5be)

In other places it is still like this:
![image](https://github.com/TTT-2/TTT2/assets/13639408/9e350c59-8238-46dc-a8e1-2a3aeac32e28)

I get this is automatically done. But I don't think it should be done this way. There are at least three different styles for hooks in this code base now

Reply:
> There are at least three different styles for hooks in this code base now

which you will never need to think about because it is autoformatted in a deterministic way :)

Reply:
Just because it is done autoamtically doesn't mean that it is done good. There should only be the first and the last case, Also does the beautifier understand the syntax? What happens when I add more/longer arguments to case 2? Judging by this:
![image](https://github.com/TTT-2/TTT2/assets/13639408/460ae3ca-3f30-40f4-8b4d-5c6d7848b1de)

I guess it would be:
```lua
local role, team =
	hook.Run(
		"hookname",
		activator,
		self,
		activator,
		caller,
		data
	)
```

Do In understand this correctly?

Reply:
serious question: do you want to think about code style?

i certainly don't. i'd rather think about the code itself

Reply:
I thought this whole PR is about code style? But yes, I aldready use style 1 and 3 (and never 2) when manually indenting code because I think style 2 and 4 look ugly.

Reply:
> I thought this whole PR is about code style?

yes code style without arbitrary choices that anyone can replicate by running an automation tool. This lessens mental load as well as eliminates unneeded changes and prevents our reviews from having random styling nits in them.

> But yes, I aldready use style 1 and 3 (and never 2) when manually indenting code because I think style 2 and 4 look ugly.

"manually indenting" oO why would you do that to yourself?
and i have to iterate: this is not about looks

Reply:
> "manually indenting" oO why would you do that to yourself?

I mean I decide whether I use
```lua
bla(a, vb, c)
```
or
```lua
bla(
	a, 
	b, 
	c
)
```

> and i have to iterate: this is not about looks

I might be alone with this - but to me it is also about looks. Code should look clean

Reply:
> Code should look clean

at the expense of having to constantly think about how to format stuff?

Reply:
I'm not really thinking about it, it is like walking. I'm not thinking about how I have to move my legs. But aren't we able to configure the tool in a way to set how we want our code to look like?

Reply:
> I'm not really thinking about it, it is like walking. I'm not thinking about how I have to move my legs.

I'm sorry but this is 1. not remotely comparable, just think about how some people have an inner monologue and the others don't. and 2. this is a super privileged way of thinking

> But aren't we able to configure the tool in a way to set how we want our code to look like?

Only somewhat as it is an opinionated formatter that is easily available for VSCode, Sublime Text and Neovim or any other editor as it is a standalone program.

If you want to program an alternative, by all means do it.